blocks/init/src/Blocks/custom/tabs/tabs.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * Template for the Tabs Block.
+ *
+ * @package EightshiftBoilerplate
+ */
+
+use EightshiftBoilerplateVendor\EightshiftLibs\Helpers\Components;
+
+$manifest = Components::getManifest(__DIR__);
+
+$blockClass = $attributes['blockClass'] ?? '';
+$blockJsClass = $manifest['blockJsClass'] ?? '';
+$tabJsClass = $manifest['tabJsClass'] ?? '';
```

Comment:
all described and all comments form tab.php applies. JS and PHP must be 99% the same

Reply:
Regarding this, I've decided for somewhat different logic on the frontend compared to the editor with the reasoning that it makes sense for individual tab attributes (such as title) to be stored exclusively as attributes of the Tab block. The activity status of a tab is determined from the Tabs-level state (as it makes sense to do so in React, and as it avoids direct DOM traversal in Gutenberg), while on the frontend as the only child of the Tabs block with a `is-visible-tab` class. Hope that's okay - let me know.